tests/reftests/pin.unix.test

```diff
@@ -0,0 +1,103 @@
+N0REP0
+### : depext update :
+### OPAMYES=1
+### opam option depext-run-installs=false
+Set to 'false' the field depext-run-installs in global configuration
+### <pin:bar/bar.opam>
+opam-version: "2.0"
+depends: "qux"
+depexts: [ "inexistant" ]
+### <add_pin_depends.sh>
+echo "pin-depends: [ \"qux.dev\" \"file://$BASEDIR/qux\" ]" >> bar/bar.opam
+### sh add_pin_depends.sh
+### <pin:qux/qux.opam>
+opam-version: "2.0"
+depexts: [ "another-inexistant" ]
+### opam switch create pinning --empty
+### opam pin ./bar | '(apt-get|brew|port)' -> 'syspkgmanager'
+Package bar does not exist, create as a NEW package? [y/n] y
+The following additional pinnings are required by bar.dev:
+  - qux.dev at file://${BASEDIR}/qux
+Pin and install them? [y/n] y
+Package qux does not exist, create as a NEW package? [y/n] y
+[qux.dev] synchronised (no changes)
+qux is now pinned to file://${BASEDIR}/qux (version dev)
+bar is now pinned to file://${BASEDIR}/bar (version dev)
+
+The following actions will be performed:
+=== install 2 packages
+  - install bar dev (pinned)
+  - install qux dev (pinned)
+
+The following system packages will first need to be installed:
+    another-inexistant inexistant
+
+<><> Handling external dependencies <><><><><><><><><><><><><><><><><><><><><><>
+
+This command should get the requirements installed:
+
+    syspkgmanager install another-inexistant inexistant
+
+Would you like opam to:
+> 1. Check again, as the package is now installed
+  2. Attempt installation anyway, and permanently register that this external dependency is present, but not detectable
+  3. Abort the installation
+
+[1/2/3] 1[D
[K[1/2/3] 1
```

Comment:
Just to remember to have a look at it, this line is suspicious and should not happen (cf. Louis)